### DISCUSSION TITLE
Para um Front-end em início de carreira, é interessante estudar mais UI/UX ou mais back-end?

### DISCUSSION THREAD

**Discussion by leoamrl** [frontendbr/forum#1015] [Discussão]
Acho que a pergunta ficou meio vaga, mas explicando:
Um profissional Front-end Pleno (domina HTML, CSS, JS, se vira com Sketch e PS, e tem noções de back e banco de dados) que deseja agregar mais para a sua carreira deveria voltar seus estudos pro lado de UI/UX ou alguma linguagem back-end?
Suponha que só seja possível estudar uma delas.

**Comment by felipefialho** [comment]
Linkando algumas issues: #32 #415 #815 #75 #114 #73

**Comment by tunnes** [comment]
Cara acho isso bem relativo... trampo sobre o título de senior (mas ainda não me considero um), sempre que tenho um tempo livre tento estudar algo distante do meu cotidiano, ai varia de funcional com haskell, UML, MER até tecnicas de gestão de projetos e por incrível que pareça quase sempre alguma coisa que vi nessa bobeira reflete em algo do meu cotidiano, como o gerenciamento de estados do Redux que já tinha visto o conceito quando parei pra ver o básico de funcional com HS.

Sobre as ferramentas é bacana tu pegar alguma linguagem server side (recomendo Ruby ou PHP) isso da uma noção incrível sobre o fluxo completo das aplicações e outras ferramentas você acaba pegando com o tempo, necessidades sazonais ou com um par que te passa uma carga diferente. 

Enfim acho que sempre é valido lembrar que na estrada do saber é facil a cobiça pelo conhecimento lhe levar a lugar algum, então sempre mantenha um workflow confortável e atual..

**Comment by Fepame** [comment]
@tunnes Sensacional a sua resposta! 
Eu adicionaria node nas linguagens server side.
